src/shadowbox/server/shared_metrics.ts

```diff
@@ -233,11 +236,15 @@ export class OutlineSharedMetricsPublisher implements SharedMetricsPublisher {
   }
 
   private async reportFeatureMetrics(): Promise<void> {
+    const keys = this.keyConfig.data().accessKeys;
     const featureMetricsReport = {
       serverId: this.serverConfig.data().serverId,
       serverVersion: version,
       timestampUtcMs: this.clock.now(),
-      dataLimit: {enabled: !!this.serverConfig.data().accessKeyDataLimit},
+      dataLimit: {
+        enabled: !!this.serverConfig.data().accessKeyDataLimit,
+        perKeyLimitCount: keys?.reduce((count, next) => count + (next.dataLimit ? 1 : 0), 0) || 0
```

Comment:
nit: may be more readable to extract the counting to a separate function.

Reply:
Done